### PR TITLE
Added Rekognition Entity

### PR DESCRIPTION
**Pull request by aj-NR**
### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an explanation above.

### CODE REVIEW

File: definitions/infra-awsrekognition/summary_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+requests:
+  goldenMetric: requests
+  title: Total Request Count
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/summary_metrics.yml]: For consistency, please lowercase the word "Count".

File: definitions/infra-awsrekognition/summary_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  unit: COUNT
+userError:
+  goldenMetric: userError
+  title: User Error count
+  unit: COUNT
+runtimeUserErrors:
+  goldenMetric: runtimeUserErrors
+  title: RuntimeUserErrors count
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/summary_metrics.yml]: For consistency, the title should be "Runtime User Errors count"

File: definitions/infra-awsrekognition/golden_metrics.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  title: User Error count
+  unit: COUNT
+  queries:
+    aws:
+      select: sum(aws.rekognition.UserErrorCount)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+runtimeUserErrors:
+  title: Server Error Count
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/golden_metrics.yml]: For consistency, the word "Count" should be lowercase

File: definitions/infra-awsrekognition/golden_metrics.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+requests:
+  title: Total Request Count
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/golden_metrics.yml]: For consistency, the word "Count" should be lowercase

File: definitions/infra-awsrekognition/dashboard.json [hunk 5]
```diff
@@ -0,0 +1,125 @@
+{
+  "name": "AWS Rekognition",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS Rekognition",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Request Counts",
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/dashboard.json]: For consistency, the title should be "Request count"

File: definitions/infra-awsrekognition/dashboard.json [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                "query": "FROM Metric SELECT count(aws.rekognition.CallCount) as `Total Requests`, count(aws.rekognition.SuccessfulRequestCount) as `Successful Requests`, count(aws.rekognition.ThrottledCount) as `Throttled Count`"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Errors Count",
```
Comment (glennthomas1) [inline definitions/infra-awsrekognition/dashboard.json]: For consistency, the title should be "Errors count"

**Review by glennthomas1 — CHANGES_REQUESTED** [review]
A few consistency changes needed.

**Review by glennthomas1 — CHANGES_REQUESTED** [review]
A few consistency changes needed.

File: definitions/infra-awsrekognition/summary_metrics.yml (hunk 1 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/summary_metrics.yml]: addressed

File: definitions/infra-awsrekognition/summary_metrics.yml (hunk 2 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/summary_metrics.yml]: addressed

File: definitions/infra-awsrekognition/golden_metrics.yml (hunk 3 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/golden_metrics.yml]: adderssed

File: definitions/infra-awsrekognition/golden_metrics.yml (hunk 4 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/golden_metrics.yml]: addressed

File: definitions/infra-awsrekognition/dashboard.json (hunk 5 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/dashboard.json]: addressed

File: definitions/infra-awsrekognition/dashboard.json (hunk 6 above)
Comment (aj-NR) [reply definitions/infra-awsrekognition/dashboard.json]: addressed

**Review by glennthomas1 — APPROVED** [review]
Looks good!

**Pull request merged**